packages/embed/src/factories/create-widget/widget-options.ts

```diff
@@ -3,5 +3,22 @@ import { ActionableOptions, BaseOptions, UrlOptions } from '../../base'
 export type WidgetOptions = BaseOptions &
   UrlOptions &
   ActionableOptions & {
+    /**
+     * Element to place the widget into.
+     *
+     * @type {HTMLElement}
+     */
     container: HTMLElement
+    /**
```

Comment:
Maybe these should be extracted into a type, like `SizeableOptions` ?

Reply:
hmmm good suggestion, on it